src/views/Team/index.view.tsx

```diff
@@ -1,8 +1,35 @@
 import * as React from "react";
+import MLHBanner from "components/MLHBanner/index.view";
+import Navbar from "components/Navbar/index.view";
+import Hero from "./components/Hero/index.view";
+import Card from "./components/Card/index.view";
+import Footer from "./components/Footer/index.view";
+
+import { navbarProps } from "views/Home/props/navbar.js";
+import { members } from "./props/members";
+
 import "./Team.scss";
 
+const description =
+  "Meet our team of creators, thinkers, strategists, and believers, collaborating together to create something amazing. ";
 const TeamPageView: React.FC = () => {
-  return <div className="TeamPage">Hello from TeamPageView</div>;
+  return (
+    <>
+      <MLHBanner />
+      <div className="TeamPage">
+        <Navbar page={"Team"} {...navbarProps} />
+        <Hero description={description} />
+        <div className="TeamPage__cardsSection">
+          <div className="TeamPage__cardsContainer">
+            {Object.entries(members).map(([memberName, props]) => (
+              <Card memberName={memberName} {...props} key={memberName} />
+            ))}
+          </div>
+          <Footer />
+        </div>
```

Comment:
```suggestion
        </div>
      <Footer />
```
Similar structuring replication.

Reply:
Logically, this is the way we'd want to do it; however, because of the structure of the page, I have to nest it within the `cardsSection` to give the effect of the gradient that "starts" at the top of the page. If you take a look at the `cardsSection` that is where the gradient starts and now on the entire `TeamPage`.

Reply:
Ohh, I understand that ~ we could partition the gradient background as a separate component within the `Backdrop`, but then we'd have to partition worry about height and scale. Sounds good though!